core-plugins/docs/RowDenormalizer-batchaggregator.md

```diff
@@ -43,22 +30,35 @@ example, input records have columns 'id', 'attribute', 'value' and the 'value' c
 **numPartitions:** Number of partitions to use when grouping data. If not specified, the execution framework will
 decide on the number to use.
 
+Conditions
+----------
+In case a field value is not present, then it will be considered as NULL.
+
+For Example,
+
+If keyfield('id') in the input record is NULL, then that particular row will not be considered.
```

Comment:
'will not be considered' is ambiguous. You can say 'then that particular record will be filtered out.'